test/controller/remove.document.controller.spec.unit.ts

```diff
@@ -1,22 +1,23 @@
+import { OBJECTIONS_CONFIRM_COMPANY, OBJECTIONS_DOCUMENT_UPLOAD } from "../../src/model/page.urls";
+
 jest.mock("ioredis");
-jest.mock("../../../src/middleware/authentication.middleware");
-jest.mock("../../../src/middleware/session.middleware");
-jest.mock("../../../src/middleware/objection.session.middleware");
-jest.mock("../../../src/services/objection.session.service");
-jest.mock("../../../src/services/objection.service");
+jest.mock("../../src/middleware/authentication.middleware");
+jest.mock("../../src/middleware/session.middleware");
+jest.mock("../../src/middleware/objection.session.middleware");
+jest.mock("../../src/services/objection.session.service");
+jest.mock("../../src/services/objection.service");
 
 import { Session } from "ch-node-session-handler";
 import { NextFunction, Request, Response } from "express";
 import request from "supertest";
-import app from "../../../src/app";
-import { OBJECTIONS_SESSION_NAME } from "../../../src/constants";
-import authenticationMiddleware from "../../../src/middleware/authentication.middleware";
-import objectionSessionMiddleware from "../../../src/middleware/objection.session.middleware";
-import sessionMiddleware from "../../../src/middleware/session.middleware";
-import { OBJECTIONS_DOCUMENT_UPLOAD } from "../../../src/model/page.urls";
-import * as pageURLs from "../../../src/model/page.urls";
-import { deleteAttachment, getAttachment } from "../../../src/services/objection.service";
-import { COOKIE_NAME } from "../../../src/utils/properties";
+import app from "../../src/app";
+import { OBJECTIONS_SESSION_NAME } from "../../src/constants";
+import authenticationMiddleware from "../../src/middleware/authentication.middleware";
+import objectionSessionMiddleware from "../../src/middleware/objection.session.middleware";
+import sessionMiddleware from "../../src/middleware/session.middleware";
+import * as pageURLs from "../../src/model/page.urls";
```

Comment:
there is one import imorting specifics from page.urls and one importing * from page urls. Probably best to use one or the other? Personally I would prefer only importing specifics

Reply:
should be fixed now, i blame the merge